keras_retinanet/utils/anchors.py

```diff
@@ -174,6 +180,9 @@ def bbox_transform(anchors, gt_boxes):
     targets = np.stack((targets_dx, targets_dy, targets_dw, targets_dh))
     targets = targets.T
 
+    for i in range(4):
+        targets[:, i] = (targets[:, i] - mean[i]) / std[i]
+
```

Comment:
I don't think this loop is needed here, numpy broadcasting rules cover this situation. This works just fine:
```
targets = (targets - mean) / std
```

Reply:
I'll try it out, thanks!